Cargo.toml

```diff
@@ -120,7 +120,7 @@ path = "./node/tcp"
 version = "=2.2.7"
 
 [target.'cfg(all(target_os = "linux", target_arch = "x86_64"))'.dependencies]
-tikv-jemallocator = "0.5"
+tcmalloc = { git = "https://github.com/bhansconnect/tcmalloc-rs", branch = "master" }
```

Comment:
we might want to pin to a specific SHA; we are publishing the lockfile, but just to be on the safe side; also, that particular repo uses `gperftools = 2.9.`, while the current version is `2.15`; perhaps it's worth forking?

Reply:
As discussed before, it's not really that important (as it uses the system libtcmalloc.so, but it might be in the future, or the fork I was using goes away. So, yeah, I'll fork it.

Reply:
In fact, ubuntu has 2.9.1 for `libgoogle-perftools-dev` up until the next LTS (24.04) which is not out yet, so I guess best fix on 2.9.1 for now.

Reply:
do they not have a release on crates.io?

also, i think it's a non-starter if we have to install a new binary

Reply:
There is a release, but it's 5 years old.

Reply:
What are the main changes in the last 5 years? If there were material security fixes, then we need to use the remote instead of the release. But regardless, if one needs to install something extra, this may be a nonstarter.